_quarto.yml

```diff
@@ -67,10 +67,13 @@ format:
     #  message: false
```

Comment:
In the end (not necessarily now) I'd like to clean up this file and remove options we don't use anyway. We won't ever convert this report into PDF, so why keep the options :)?